docs/table/performance.inc.rst

```diff
@@ -39,3 +39,23 @@ then use ``serialize_method='data_mask'``.
 It uses the non-masked version of data and it is faster::
 
     >>> tm.write('tm.ecsv', overwrite=True, serialize_method='data_mask') # doctest: +SKIP
+
+Reading a ``.fits`` in |Table| using
+:meth:`~astropy.table.Table.read` when ``memmap=False`` can be very slow
+because it reads full |Table| in the memory ::
+
+    >>> import numpy as np
+    >>> from astropy.table import Table
+    >>> tbl = Table({
```

Comment:
Should we skip this too? CI has limits.

Reply:
agreed with @pllim. For this example I think adding `.. doctest-skip::` directive in front of the example would work well. See example in e.g. docs/nddata/utils.rst